src/linker/Linker.Steps/ResolveFromXmlStep.cs

```diff
@@ -434,6 +434,8 @@ void MarkMethod (MethodDefinition method)
 				Context.LogMessage ($"Duplicate preserve in {_xmlDocumentLocation} of {method.FullName}"); 
 
 			Annotations.Mark (method);
+			if (Context.AnnotateUnseenCallers)
```

Comment:
Why is it needed?

Reply:
The methods named here are methods that may have call sites that we do not see, or they may be reflected.